extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -60,13 +60,22 @@ describes.realWin(
 
     async function shoppingDataDispatchStoreService() {
       const shoppingData = {
-        'sunglasses': {'productTitle': 'Spectacular Spectacles'},
+        'page1': {'sunglasses': {'productTitle': 'Spectacular Spectacles'}},
       };
       storeService.dispatch(Action.ADD_SHOPPING_DATA, shoppingData);
     }
 
-    it('should build and layout shopping tag component', () => {
+    async function createShoppingTag() {
+      shoppingTag.element.setAttribute('data-product-id', 'sunglasses');
+      await shoppingDataDispatchStoreService();
+
+      expect(() => shoppingTag.buildCallback()).to.not.throw();
       expect(() => shoppingTag.layoutCallback()).to.not.throw();
+    }
+
+    it('should build and layout shopping tag component', async () => {
+      await createShoppingTag();
+      expect(shoppingTag.shoppingTagEl_).to.be.not.null;
```

Comment:
I don't actually see a `shoppingTagEl_` property being declared in the constructor of `amp-story-shopping-tag`. Instead, it appears that the property is being declared and initialized within `createAndAppendInnerShoppingTagEl_()`. Please declare the property in the constructor, presumably with an initial value of `null`